api/openapi.yaml

```diff
@@ -1516,6 +1516,10 @@ components:
         launchTime:
           type: string
           format: date-time
+        rootVolumeSizeGB:
+          type: integer
+        rootVolumeEncrypted:
+          type: boolean
```

Comment:
What do you think about nesting this info in a new object to avoid the prefixing? like:

```
rootVolume:
   sizeGB:
      type: integer
   encrypted:
      type: boolean
 ```
 ? 
 
 Do you think we should pull all the disk info off the instance? Like:
 
 ```
 disks:
    type: array
    items:
       type: object
       properties:
          rootVolume:
             description: This disk is the root volume, only one disk is allowed to have this marked true.
             type: boolean
          sizeGB:
              type: integer
          encrypted:
              type: boolean
 ```

Reply:
I can move it to a new object, but not sure that that's the time to make it a list, i think that it will block the UI as we need a UX for that. We can do it when there will be a need

Reply:
👍